src/main/kotlin/no/nav/syfo/identhendelse/kafka/IdenthendelseConsumerService.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.syfo.identhendelse.kafka
+
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.delay
+import no.nav.syfo.application.ApplicationState
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import kotlin.time.Duration.Companion.seconds
+
+class IdenthendelseConsumerService(
+    private val kafkaConsumer: KafkaConsumer<String, KafkaIdenthendelseDTO>,
+    private val applicationState: ApplicationState,
+) {
+    suspend fun startConsumer() = coroutineScope {
+        kafkaConsumer.subscribe(listOf(PDL_AKTOR_TOPIC))
+        log.info("Started consuming pdl-aktor topic")
+        while (applicationState.ready) {
+            try {
+                val records = kafkaConsumer.poll(Duration.ofSeconds(POLL_DURATION_SECONDS))
+                if (records.count() > 0) {
+                    records.forEach { record ->
+                        log.info(
+                            """
+                            Consumed pdl-aktor topic with data
+                            key: ${record.key()}
```

Comment:
Mulig vi bør være forsiktige med å logge key/value i prod hvis de inneholder fødselsnummer 🤔

Reply:
Godt poeng! Målet er å få kjørt en dryrun i dev og se hva som kommer inn. Men skal få fjernet den loggingen før prod tenker jeg, ja 👌🏼

Reply:
Lurt med dry-run 👌 Kan evt lage en toggle for å styre at man ikke konsumerer i prod til å begynne med.